config/environments/development.rb

```diff
@@ -36,11 +36,12 @@
   config.action_mailer.raise_delivery_errors = false
   config.action_mailer.perform_caching = false
   config.action_mailer.delivery_method = :smtp
-  config.action_mailer.smtp_settings = { :address => "localhost", :port => 1025 }
+  config.action_mailer.smtp_settings = { address: "localhost", port: 1025 }
   config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
   config.action_mailer.asset_host = 'http://localhost:3000'
   config.action_controller.asset_host = 'localhost:3000'
-  config.active_job.queue_adapter = :inline
+  Rails.application.routes.default_url_options[:host] = 'localhost'
```

Comment:
just `routes.default_url_options[:host] = 'localhost'`, you do not need `Rails.application` because you are writing it inside configuration block